app/assets/v2/js/pages/profile_tokens.js

```diff
@@ -70,6 +72,8 @@ async function buyPToken(tokenAmount) {
     pTokenAddress // TODO: this needs to be derived from the profile page
   );
 
+  // TODO add token approval step
```

Comment:
Will these TODOs be addressed in a separate PR?

Reply:
Yea this TODO is for purchasing ptokens, whereas this PR is only for deploying them. So that will be addressed separately